src/local/room_position.rs

```diff
@@ -263,8 +260,8 @@ impl Position {
     pub fn from_packed(packed: i32) -> Self {
```

Comment:
Looking at the types, is there a reason this is an i32?

Reply:
Not as far as I know, @shanemadden would have more insight.

Reply:
Huh! That's pulled straight forward from the current master branch unaltered, but you're right, that _shouldn't_ work. We have tests using negative numbers, but I guess they're not correct packed positions? Will dig a bit more and confirm.